src/main/java/com/teachMeSkills/task/servlet/ServletSaveRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.teachMeSkills.task.servlet;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/save-request")
+public class ServletSaveRequest extends HttpServlet {
+

```

Comment:
How do you open your jsp with inputs without doGet method?

Reply:
done